setup.py

```diff
@@ -49,6 +50,17 @@
 ]
 
 
+if "--skip-npm" in sys.argv:
+    print("Skipping install of webtools as requested.")
+    skip_npm = True
+    sys.argv.remove("--skip-npm")
+elif not find_executable("npm"):
+    print("No npm found, skipping install of webtools.")
+    skip_npm = True
```

Comment:
We should be careful about this - sdist should fail if npm is unavailable. Simiarly, install should fail if npm is unavailable and built js isn't present.